src/prefect/cli/deploy.py

```diff
@@ -250,6 +258,7 @@ async def deploy(
                 actions=actions,
                 deploy_all=deploy_all,
                 ci=ci,
+                build_push_once=build_push_once,
```

Comment:
The option is passed into `prefect deploy` here, which passes it to `_run_multi_deploy` at line 686